src/main/java/org/fundacionjala/coding/jose/FizzBuzz.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.coding.jose;
+
+/**
+ * Created by JoseTorrez on 8/25/2017.
+ */
+public final class FizzBuzz {
+  /**
+   * Default Constructor.
+   */
+  private FizzBuzz() {
+
+  }
+
+  /**
+   * @param number Integer received.
+   * @return String word or number.
+   */
+  public static String numberDivisor(Integer number) {
+    String variable = "";
+    while (number > 0) {
+      if (((number % 3 == 0) && (number % 5 == 0)) || (Integer.toString(number).contains("3")
```

Comment:
Remove the unnecessary parenthesis in all your if conditional
Add constants for 3 and 5
also for Fizz and Buzz